src/main/java/reposense/report/CommitReportJson.java

```diff
@@ -24,7 +24,7 @@
     /**
      * Empty constructor to construct an empty commit report.
      */
-    public CommitReportJson() {
+    public CommitReportJson(String authorName) {
```

Comment:
Instead of `authorName`,  I prefer `displayName`. Can ask seniors' opinion.

Reply:
Yes, display name is more logically correct.